apps/gelato-vrf/typechain-types/GelatoVRFConsumerBase.ts

```diff
@@ -0,0 +1,163 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Typechain files are typically gitignored and included as part of the build process

Reply:
Indeed, it is created with each new deployment, which is why the folder has been deleted.